virttest/remote_commander/remote_runner.py

```diff
@@ -703,6 +711,54 @@ def exit(self):
         self._exit = True
         return "bye"
 
+    def run_test(self, test_path):
+        """Call run() function at external python module.
+
+        Parameters
+        ----------
+        test_path : str
+            Path to python file. Call run() function at this module.
+
+        Returns
+        -------
+            module.run().return
+        """
+        assert os.path.isfile(test_path)
+        module = imp.load_source('ext_test', test_path)
```

Comment:
I can modify tests. I am really sorry, but I do not understand. We are talking about the place where we should call a function and we must provide some arguments to it. Predefined arguments. The test are aware that they are called from remote_commander, and they expect callback object.